substrate/client/consensus/beefy/src/keystore.rs

```diff
@@ -111,7 +111,15 @@ impl<AuthorityId: AuthorityIdBound> BeefyKeystore<AuthorityId> {
 				sig_ref.to_vec()
 			},
 
-			_ => Err(error::Error::Keystore("key type is not supported by BEEFY Keystore".into()))?,
+			_ => store
```

Comment:
I am not replacing previous 2 cases with this generic call, due they perform some additional hashing, which will be not done in case I will remove them